examples/install_requirements.py

```diff
@@ -61,12 +76,13 @@ def hasWhitespace(string):
 if sys.version_info[0] != 3:
     raise RuntimeError("Examples require Python 3 to run (detected: Python {})".format(sys.version_info[0]))
 
-if platform.machine() == "arm64" and platform.system() == "Darwin":
+if thisPlatform == "arm64" and platform.system() == "Darwin":
     err_str = "There are no prebuilt wheels for M1 processors. Please open the following link for a solution - https://discuss.luxonis.com/d/69-running-depthai-on-apple-m1-based-macs"
     raise RuntimeError(err_str)
 
-is_pi = platform.machine().startswith("arm") or platform.machine().startswith("aarch")
-if is_pi and sys.version_info[1] == 9:
+is_pi = thisPlatform.startswith("arm") or thisPlatform.startswith("aarch")
```

Comment:
This seems to catch ARMv8 as well (eg Ubuntu ARMv8 on Pi). 
There is a manylinux support for this architecture and most major wheels have prebuilts for it (including `opencv-python`)